pysankey/tests/generic_test.py

```diff
@@ -3,14 +3,55 @@
 import os
 import unittest
 
+import pandas as pd
+
 
 class GenericTest(unittest.TestCase):
 
-    """ Generic tests for sankey, figure_name is used the resulting images
+    """ Generic tests for sankey
+
+    If figure_name is used, the resulting images
     will be removed at the end of the tests. """
 
+    @classmethod
+    def setUpClass(cls):
+        cls.figure_name = ''
+        cls.data = ''
+        cls.colorDict = ''
+
     def tearDown(self):
         path = "{}.png".format(self.figure_name)
         # Comment this to check the resulting image
         if os.path.exists(path):
             os.remove(path)
+
+
+class TestFruit(GenericTest):
```

Comment:
Nice job upping the coverage to almost 100 %!

Reply:
Thanks :)